### PR TITLE
Mark fixers as risky

### LINKED ISSUE #942: Proposal: mark fixer as "risky" (change meaning of code)
What do you think about marking fixers as risky?

Currently: psr0, strict, strict_param.
In future fixer that trim all trailing spaces (even inside string content), as PSR2 requires.

Then that fixer will run only if risky fixers are allowed by config/cli.

Then, in future, one may run all PSR2 fixers safely without changing meaning of code or run all PSR2 fixers with that risky flag and then apply all PSR2 rules, even if they change code meaning.

### PR DESCRIPTION
**Pull request by keradus**
Closes #942

### CODE REVIEW

**Comment by stof** [comment]
Looks good to me

File: README.rst [hunk 1]
```diff
@@ -160,6 +160,9 @@ When using combinations of exact and blacklist fixers, applying exact fixers alo
 A combination of ``--dry-run`` and ``--diff`` will
 display a summary of proposed fixes, leaving your files unchanged.
 
+The ``--allow-risky`` option allow you to set whether riskys fixer may run. Default value is taken from config file.
```
Comment (SpacePossum) [inline README.rst]: `allows`

File: Symfony/CS/Config/Config.php [hunk 2]
```diff
@@ -195,6 +196,24 @@ public function getPhpExecutable()
     /**
      * {@inheritdoc}
      */
+    public function getRiskyAllowed()
```
Comment (SpacePossum) [inline Symfony/CS/Config/Config.php]: maybe `isRiskyAllowed()`, following http://symfony.com/doc/current/contributing/code/conventions.html?

**Comment by SpacePossum** [comment]
looking good :)

File: Symfony/CS/Config/Config.php (hunk 2 above)
Comment (keradus) [inline Symfony/CS/Config/Config.php]: can't see `isFoo`, only `isEmptyFoo`

File: README.rst (hunk 1 above)
Comment (keradus) [inline README.rst]: yeah ;)

File: Symfony/CS/Config/Config.php (hunk 2 above)
Comment (SpacePossum) [inline Symfony/CS/Config/Config.php]: fair enough :+1:  :)

**Pull request merged**